### PR TITLE
Allows for saving a rendered plot

### PR DESCRIPTION
**Pull request by jeffreypaul15**
This method allows a user to save a particular scene with `.vtm` extension (multiple meshes).
The file is stored in `filename.vtm`.

### CODE REVIEW

File: sunkit_pyvista/plotter.py [hunk 1]
```diff
@@ -170,3 +174,19 @@ def plot_field_lines(self, field_lines, **kwargs):
...
+        """
+        Saves a given plot to a vtm file.
+
+        Parameters
+        ----------
+        filename : `str`
+            Name of the file to save as, should end with vtm.
+        """
+        all_meshes = pv.MultiBlock(self.mesh)
+        if filename.endswith('vtm'):
```
Comment (nabobalis) [inline sunkit_pyvista/plotter.py:189]: Do we just want to tell them we will append the extension in the doc string and do away with this?

Comment (jeffreypaul15) [reply sunkit_pyvista/plotter.py:189]: Sure that would work too I suppose

**Comment by nabobalis** [comment]
What about loading?

**Comment by jeffreypaul15** [comment]
> What about loading?

Yeah I was caught up with trying to get the colors to save, right now unless we specify a specific cmap while plotting, the colors won't save. 
I'll add the loading bit after I figure this out.

**Pull request closed without merging**